.flake8

```diff
@@ -0,0 +1,11 @@
+[flake8]
```

Comment:
This .flake8 configuration file is the one from https://github.com/psf/black, which makes flake8 not complain about code formatted by black.

Reply:
Could you add these also to `meeshkan-codestyle` repo so it's consistent across repos? Thanks!

Reply:
👍Adding in https://github.com/Meeshkan/meeshkan-codestyle/pull/2